app/src/main/java/de/westnordost/streetcomplete/data/QuestController.java

```diff
@@ -257,6 +267,53 @@ else if(quest.getStatus() == QuestStatus.CLOSED)
 		});
 	}
 
+	private boolean solveCompleteQuest(long questId, Bundle bundle)
+	{
+		CompleteQuest q = completeQuestDB.get(questId);
+
+		String answer;
+
+		switch (q.getComplete().completeType)
+		{
+			case CompleteTypes.CHART:
+				Boolean value = bundle.getBoolean(CompleteQuestType.ANSWER);
+				answer = value ? "Yes" : "No";
+				break;
+
+			case CompleteTypes.TRANSLATION:
+				answer = bundle.getString(CompleteQuestType.ANSWER);
```

Comment:
Did you already implement the things for a translation quest? As you do not have an example quest for that…

Reply:
No, not yet, because I don't have a real use case for that... Do you maybe have one?

Reply:
@westnordost said on https://github.com/westnordost/StreetComplete/issues/633#issuecomment-334843714:
> "How do you call this type of shop in [some language]?"

So basically it would be for contributing data for https://github.com/ENT8R/tags2names, in order to do https://github.com/westnordost/StreetComplete/issues/309.

Reply:
I already thought about that too... But I think that this won't work... Just imagine the amount of data which is needed for this dictionary. Then there would be more translation quests than normal OSM-related quests...

Reply:
You do not have to show a quest for each element on the map. E.g. you can just hide 2/3 of the complete quests to show them more rarely.